Generic_Plugin.php

```diff
@@ -84,6 +84,31 @@ function run() {
 					'ob_callback'
 				) );
 		}
+
+		// If WP < 5.5, disable jquery-migrate on the front-end, if configured.
+		if ( ! is_admin() && version_compare( get_bloginfo( 'version' ), '5.5', '<' ) && $this->_config->get_boolean( 'jquerymigrate.disabled' ) ) {
+			add_action( 'wp_default_scripts', array( $this, 'disable_jquery_migrate' ) );
+		}
+	}
+
+	/**
+	 * Disable jquery-migrate.
```

Comment:
why

Reply:
It was a feature request that was marked Next.